packages/sol-tracing-utils/test/sol_compiler_artifact_adapter_test.ts

```diff
@@ -16,6 +16,7 @@ describe('SolCompilerArtifactAdapter', () => {
             const contractsData = await zeroExArtifactsAdapter.collectContractsDataAsync();
             _.forEach(contractsData, contractData => {
                 expect(contractData).to.have.keys([
+                    'name',
```

Comment:
Was this failing before?

Reply:
Yeah, it was failing locally for me. Haven't seen it fail on CI though because I fixed it before pushing. I thought it might have been different version of mocha, but even after blitzing node_modules and reinstalling everything it still failed.